src/analysis/stack-lattice.h

```diff
@@ -20,14 +68,13 @@ template<typename StackElementLattice> class StackLattice {
     : stackElementLattice(stackElementLattice) {}
 
   class Element {
-    // The top lattice is
+    // The top lattice can be imagined as an infinitely high stack, which
+    // is unreachable in most cases. In practice, we make the stack an
+    // optional, and if we are the top lattice, the stack doesn't exist in
+    // the optional signalling the top lattice.
```

Comment:
editing just for clarity
```suggestion
    // optional, and we represent top with the absence of a stack.
```